extension/experiments/voice/api.js

```diff
@@ -54,6 +54,15 @@ this.voice = class extends ExtensionAPI {
           async quitApplication() {
             return runCommand("cmd_quitApplication");
           },
+
+          async openPreferences() {
+            return runCommand("Browser:OpenAboutContainers");
```

Comment:
This goes to the same thing as "Manage Containers", which is technically in preferences, but not a great solution. 

It works for me if this calls `return runCommand("menu_preferences");` which points to a menuitem (but works just as well as a `<command>`): https://searchfox.org/mozilla-central/source/browser/base/content/browser-menubar.inc#136

Reply:
I have this fixed. So, if I wanted to use this as `<command>`, how will I implement it? What will it look like in the code?